documentation/tools/dataprotector/methods/processprotecteddata.md

```diff
@@ -58,6 +61,8 @@ A set of URLs representing the input files required for application execution.
 
 A set of requester secrets necessary for the application's execution. This is represented as a mapping of numerical identifiers to corresponding secrets stored in the secrets manager needed for the application's execution.
 
+Secrets are accessible during the application's execution with environment variables `IEXEC_REQUESTER_SECRET_n`.
```

Comment:
What's the weird `_n` at the end of the constant?
I'm not sure I understand this correctly.

Does it refer to this? https://protocol.docs.iex.ec/for-developers/confidential-computing/access-confidential-assets/requester-secrets

If so, can we add a link to this page?

Example:

"For more details, see [Requester secrets](https://protocol.docs.iex.ec/for-developers/confidential-computing/access-confidential-assets/requester-secrets)."